kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -16,17 +23,30 @@
   import NotificationsRoot from 'kolibri.coreVue.components.NotificationsRoot';
   import plugin_data from 'plugin_data';
   import { PageNames } from '../constants';
+  import PostSetupModalGroup from '../../../../device/assets/src/views/PostSetupModalGroup.vue';
+
+  const welcomeDimissalKey = 'DEVICE_WELCOME_MODAL_DISMISSED';
 
   export default {
     name: 'LearnIndex',
     components: {
       NotificationsRoot,
+      PostSetupModalGroup,
     },
     computed: {
       ...mapGetters(['isUserLoggedIn']),
+      ...mapState({
+        welcomeModalVisibleState: 'welcomeModalVisible',
+      }),
       ...mapState({
         loading: state => state.core.loading,
       }),
+      welcomeModalVisible() {
+        return (
+          this.welcomeModalVisibleState &&
+          window.sessionStorage.getItem(welcomeDimissalKey) !== 'true'
```

Comment:
I don't know if it matters in this case, but for reactive usage of session storage, you can do this: https://vueuse.org/core/useSessionStorage/

Come to think of it, that might remove the need to separately track in vuex if we just use this instead.